pkg/integrations/v2/metricsutils/versionshim.go

```diff
@@ -15,20 +15,13 @@ import (
 	"github.com/grafana/agent/pkg/util"
 )
 
-// CreateShim creates a shim between the v1.Config and v2.Config. The resulting
-// config is NOT registered.
-func CreateShim(before v1.Config, common common.MetricsConfig) (after v2.UpgradedConfig) {
-	return &configShim{orig: before, common: common}
-}
```

Comment:
Should this be removed at this stage? We might need it later

Reply:
If needed later, then I can re-add. If they are needed then they will likely take on a slightly different form.